src/i18n/i18nConfig.ts

```diff
@@ -30,9 +30,6 @@ i18n
     //debug: true,
     fallbackLng: "en",
     resources,
-    interpolation: {
```

Comment:
We don't want this anymore?  
Here it looks like we might want this? https://react.i18next.com/legacy-v9/step-by-step-guide#set-the-global-wait-option

Reply:
Or is this just legacy?

Reply:
Hm, I guess we don't need to set this anymore.

Reply:
The docs are a bit confusing for this library admittedly, I'm guessing they suggest turning it off for some sort of optimization purpose and because it's "not needed" for React, but I see no reason not to if there's any chance it can make things safer/makes the code cleaner and if there's no downside. That said I can add it back if there's something I'm missing!

Reply:
I saw your comment and the research after I wrote this. All good. Thanks a lot for your efforts!

Reply:
Happy to 😄